delete-file.gpt

```diff
@@ -0,0 +1,4 @@
+Name: Knowledge File Deletion
+Description: Delete a file.
+
+#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool delete-file "${GPTSCRIPT_INPUT}"
```

Comment:
Does the input contain the dataset flag or is it set via env?

Reply:
Good question. It is set through env variable but I think I forgot to set it. I will update it in my PR